design_doc/mdeditMay22_1.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
If this is something specific for MILES, please name it in a way that makes that obvious, e.g., `design_doc/miles_defaults.xml`.